app/javascript/src/services/api/config.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+export const axiosInstance = axios.create({
+  withCredentials: true,
+  headers: {
+    "X-CSRF-TOKEN": document.querySelector('meta[name="csrf-token"]').content
+  }
+});
```

Comment:
Does this mean that we don't need `app/javascript/src/config/axios.js` file anymore?

Reply:
yes but we have to refactor other components that still uses confit/axios.js and the endpoints_api will also go out from config.js